cloudify/plugins/workflows.py

```diff
@@ -643,11 +643,11 @@ def _make_execute_operation_graph(ctx, operation, operation_kwargs,
 
 
 @workflow(resumable=True)
-def execute_operation(ctx, *args, **kwargs):
+def execute_operation(ctx, operation, *args, **kwargs):
     """ A generic workflow for executing arbitrary operations on nodes """
-
+    name = 'execute_operation_{0}'.format(operation)
```

Comment:
Can we add tests also for this?

Reply:
no, because operations are only stored when using a manager; when using local, they're not stored, and so this name is passed nowhere

that's why i added integration tests - because i couldn't test this on the unittest side